app/javascript/dashboard/routes/dashboard/settings/personal/AudioAlertCondition.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div>
+    <label
+      class="flex justify-between pb-1 text-sm font-medium leading-6 text-ash-900"
+    >
+      {{ label }}
+    </label>
+    <div class="flex flex-col gap-2">
+      <div
+        v-for="item in items"
+        :key="item.id"
+        class="flex flex-row items-start gap-2"
+      >
+        <input
+          :id="item.id"
+          v-model="item.model"
+          type="checkbox"
+          :value="item.value"
+          class="flex-shrink-0 mt-0.5 border-ash-200 border checked:border-none checked:bg-primary-600 dark:checked:bg-primary-600 shadow appearance-none rounded-[4px] w-4 h-4 focus:ring-1 after:content-[''] after:text-white checked:after:content-['✓'] after:flex after:items-center after:justify-center after:text-center after:text-xs after:font-bold after:relative"
+          @input="onChange"
+        />
```

Comment:
@muhsin-k Can we move it to a component? So we can reuse it when introducing new design changes.